<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CosyVoice</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background: #1a1412;
}

QWidget {
    background-color: transparent;
    font-family: &quot;Press Start 2P&quot;, &quot;Courier New&quot;, monospace;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="headerWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#headerWidget {
    background: #2a1d19;
}</string>
      </property>
      <layout class="QVBoxLayout" name="headerLayout">
       <property name="spacing">
        <number>16</number>
       </property>
       <property name="leftMargin">
        <number>40</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>40</number>
       </property>
       <property name="bottomMargin">
        <number>24</number>
       </property>
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="text">
          <string>COSYVOICE</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel#titleLabel {
    color: #c4a77d;
    font-size: 24px;
    font-weight: bold;
    letter-spacing: 2px;
    background: transparent;
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="subtitleLabel">
         <property name="text">
          <string>It's A feeling</string>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #6b5c52;
    font-size: 10px;
    letter-spacing: 1px;
    text-transform: uppercase;
    background: transparent;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="contentWidget">
      <property name="styleSheet">
       <string notr="true">QWidget#contentWidget {
    background: #1a1412;
}</string>
      </property>
      <layout class="QVBoxLayout" name="contentLayout">
       <property name="spacing">
        <number>24</number>
       </property>
       <property name="leftMargin">
        <number>40</number>
       </property>
       <property name="topMargin">
        <number>32</number>
       </property>
       <property name="rightMargin">
        <number>40</number>
       </property>
       <property name="bottomMargin">
        <number>32</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="buttonLayout">
         <property name="spacing">
          <number>16</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnAudioClone">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string>AUDIO CLONE</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background: #1a1412;
    color: #c4a77d;
    border: 2px solid #3d2b25;
    border-radius: 0px;
    font-size: 11px;
    font-weight: bold;
    letter-spacing: 0px;
    padding: 6px 10px;
    text-align: center;
}

QPushButton:hover {
    background: #3d2b25;
    color: #e8d5c4;
    border-color: #c4a77d;
}

QPushButton:pressed {
    background: #c4a77d;
    color: #1a1412;
    border-color: #c4a77d;
}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnModelDownload">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string>MODEL DOWNLOAD</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background: #1a1412;
    color: #c4a77d;
    border: 2px solid #3d2b25;
    border-radius: 0px;
    font-size: 11px;
    font-weight: bold;
    letter-spacing: 0px;
    padding: 6px 10px;
    text-align: center;
}

QPushButton:hover {
    background: #3d2b25;
    color: #e8d5c4;
    border-color: #c4a77d;
}

QPushButton:pressed {
    background: #c4a77d;
    color: #1a1412;
    border-color: #c4a77d;
}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnResults">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string>RESULTS</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background: #1a1412;
    color: #c4a77d;
    border: 2px solid #3d2b25;
    border-radius: 0px;
    font-size: 11px;
    font-weight: bold;
    letter-spacing: 0px;
    padding: 6px 10px;
    text-align: center;
}

QPushButton:hover {
    background: #3d2b25;
    color: #e8d5c4;
    border-color: #c4a77d;
}

QPushButton:pressed {
    background: #c4a77d;
    color: #1a1412;
    border-color: #c4a77d;
}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnStatus">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string>STATUS</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background: #1a1412;
    color: #c4a77d;
    border: 2px solid #3d2b25;
    border-radius: 0px;
    font-size: 11px;
    font-weight: bold;
    letter-spacing: 0px;
    padding: 6px 10px;
    text-align: center;
}

QPushButton:hover {
    background: #3d2b25;
    color: #e8d5c4;
    border-color: #c4a77d;
}

QPushButton:pressed {
    background: #c4a77d;
    color: #1a1412;
    border-color: #c4a77d;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="styleSheet">
          <string notr="true">QStackedWidget {
    background: transparent;
    border: none;
}</string>
         </property>
         <widget class="QWidget" name="welcomePage">
          <layout class="QVBoxLayout" name="welcomeLayout">
           <item>
            <widget class="QLabel" name="welcomeLabel">
             <property name="text">
              <string>SELECT A FEATURE TO BEGIN</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #6b5c52;
    font-size: 11px;
    letter-spacing: 0px;
    background: transparent;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="audioClonePage">
          <layout class="QVBoxLayout" name="audioCloneLayout">
           <item>
            <widget class="QLabel" name="audioClonePlaceholder">
             <property name="text">
              <string>AUDIO CLONE</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #c4a77d;
    font-size: 16px;
    font-weight: 500;
    letter-spacing: 2px;
    background: transparent;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="modelDownloadPage">
          <layout class="QVBoxLayout" name="modelDownloadLayout">
           <item>
            <widget class="QLabel" name="modelDownloadPlaceholder">
             <property name="text">
              <string>MODEL DOWNLOAD</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #c4a77d;
    font-size: 16px;
    font-weight: 500;
    letter-spacing: 2px;
    background: transparent;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="resultsPage">
          <layout class="QVBoxLayout" name="resultsLayout">
           <item>
            <widget class="QLabel" name="resultsPlaceholder">
             <property name="text">
              <string>RESULTS</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #c4a77d;
    font-size: 16px;
    font-weight: 500;
    letter-spacing: 2px;
    background: transparent;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="statusPage">
          <layout class="QVBoxLayout" name="statusLayout">
           <item>
            <widget class="QLabel" name="statusPlaceholder">
             <property name="text">
              <string>STATUS</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #c4a77d;
    font-size: 16px;
    font-weight: 500;
    letter-spacing: 2px;
    background: transparent;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">QStatusBar {
    background: #14100f;
    color: #6b5c52;
    border-top: 1px solid #2a1d19;
}

QStatusBar::item {
    border: none;
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>